### PR TITLE
[MOB-8995] Drop Manual Linking Support

### PR DESCRIPTION
**Pull request by DavidMina96**
## Description of the change

- Remove (un)linking-related files and scripts for both platforms
- Remove `hooks` property from `react-native.config.js` file
- Remove the no-longer-needed maven repo from the Sample app
- Specify a minimum `react-native` version in `peerDependencies` (0.60.0)

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
### Development
- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
### Code review 
- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,10 +23,6 @@
   },
   "homepage": "https://github.com/Instabug/instabug-reactnative#readme",
   "rnpm": {
-    "commands": {
-      "postlink": "node node_modules/instabug-reactnative/link_bridge.js",
-      "preunlink": "node node_modules/instabug-reactnative/unlink_bridge.js"
-    },
     "android": {
       "packageInstance": "new RNInstabugReactnativePackage()"
```
Comment (TheBuggedYRN) [inline package.json:27]: Is this still needed as well?

**Review by TheBuggedYRN — CHANGES_REQUESTED** [review]
We need a Changelog entry as well.

File: package.json (hunk 1 above)
Comment (a7med-mahmoud) [reply package.json:27]: I don't think so, we can remove it

**Review by a7med-mahmoud — APPROVED** [review]
Sounds good to me. I'm just curious, do we need the README file to be formatted in this PR?

**Comment by DavidMina96** [comment]
@a7med-mahmoud Not necessarily, but since I had to update already within the PR's scope, I applied the formatting changes as well.

**Review by TheBuggedYRN — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -114,36 +95,34 @@ Instabug Repro Steps are enabled by default. It captures a screenshot of each sc
...
+    set the `onStateChange` to `Instabug.onStateChange` in your NavigationContainer as follows:
 
-			```javascript
-			<NavigationContainer
-			onStateChange={  Instabug.onStateChange  }  />
-			```
+        ```javascript
+        <NavigationContainer
+        onStateChange={  Instabug.onStateChange  }  />
+        ```
```
Comment (AliAbdelfattah) [inline README.md:106]: When you display the rich diff, ` ```javascript ` is shown as part of the snippet code.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+## Unreleased
+
+* React Native 0.69 support
+* Bumps the minimum supported React Native version to 0.60.0
```
Comment (AliAbdelfattah) [inline CHANGELOG.md:4]: Please add an item for dropping the manual linking support.

**Review by TheBuggedYRN — APPROVED** [review]

**Review by AliAbdelfattah — APPROVED** [review]

**Review by a7med-mahmoud — APPROVED** [review]

**Pull request merged**